<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imgSetDialog</class>
 <widget class="QDialog" name="imgSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="realOpenButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Real...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="synthOpenButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Synth...</string>
   </property>
  </widget>
  <widget class="QComboBox" name="realComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="synthComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="similarityButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Similarity</string>
   </property>
  </widget>
  <widget class="QListView" name="similarityListView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>181</y>
     <width>256</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="realViewComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Original</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KeyPoints</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="synthViewComboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Original</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KeyPoints</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="numCumlustersEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>500</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>number of clusters</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
